README.md

```diff
@@ -32,6 +32,20 @@ You'd be running it as follows:
 touchstone_compare uperf elasticsearch ripsaw -url marquez.perf.lab.eng.rdu2.redhat.com marquez.perf.lab.eng.rdu2.redhat.com  -u 6c5d0257-57e4-54f0-9c98-e149af8b4a5c 70cbb0eb-8bb6-58e3-b92a-cb802a74bb52
 ```
 
+### Comparing on a specific identifier
+
+You can also now compare against identifiers other than the `uuid` key, so for
+example if you'd like to compare using the key `cluster_name` you can do so by
+using the `-id` argument and run as follows:
```

Comment:
@aakarshg  I've seen that the code only allows  `uuid` and `cluster_name`  as identifiers. Is this expected?, in that case the documentation should point it out correctly.

Reply:
Code allows for any identifiers, i just mentioned an example here with cluster_name, and uuid was default.

Reply:
I think not @aakarshg , the code passes the argument `choices=['uuid','cluster_name']`  to the id flag as I comment below.

Reply:
updated @rsevilla87 can you PTAL again